chapter-09/recipe-05/cxx-example/account/CMakeLists.txt

```diff
@@ -1,23 +1,30 @@
-# define minimum cmake version
-cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
+include(FetchContent)
 
-# project name and supported language
-project(account-project LANGUAGES CXX)
+FetchContent_Declare(
+  pybind11_sources
+  GIT_REPOSITORY https://github.com/pybind/pybind11.git
+  GIT_TAG        v2.2
+)
 
-# require C++11
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+FetchContent_GetProperties(pybind11_sources)
 
```

Comment:
As I have recently discovered, the `__init__.py` and `version.py` can be copied over to the installation directory by the `install` command with the following contraption:
```
set_target_properties(account
  PROPERTIES
    PREFIX "${PYTHON_MODULE_PREFIX}"
    SUFFIX "${PYTHON_MODULE_EXTENSION}"
    RESOURCE "${CMAKE_CURRENT_SOURCE_DIR}/__init__.py;${CMAKE_CURRENT_SOURCE_DIR}/version.py"
  )

install(
  TARGETS
    account
  LIBRARY
    DESTINATION ${INSTALL_LIBDIR}
    COMPONENT lib
  RESOURCE
    DESTINATION ${INSTALL_LIBDIR}
    COMPONENT lib
  )
```
Can you make this change here and other places where it might be necessary? I'll update TC accordingly. This is in line with the discussion of `PUBLIC_HEADER` later in chapter 10, which I will modify to also mention the `RESOURCE` keyword.

Reply:
We follow up on this in #431 since I would like this PR to go in to not accumulate conflicts and this good suggestion we can do as separate PR, followed by changes to TC. Otherwise I am worried that I forget to make this change TC-side.